src/coreclr/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,7 @@
+<linker>
```

Comment:
Nothing is including this new file in the project. You need to add a line like:

https://github.com/dotnet/runtime/blob/6791d05494f5aa4d1a789525c2708e775aafdc07/src/mono/System.Private.CoreLib/System.Private.CoreLib.csproj#L136